kobo/settings/base.py

```diff
@@ -199,7 +199,24 @@
     ),
     'MFA_ENABLED': (
         True,
-        'Enable two-factor authentication',
+        'Enable two-factor authentication'
+    ),
+    'MFA_I18N_HELP_TEXTS': (
+        json.dumps({
+            'default': (
+                'KoBoToolbox uses Multi-Factor Authentication. '
+                'If you no longer have access to the device registered '
+                'with your account and the backup codes, the KoBo Support Team '
+                'can assist you regaining access to the system.'
+                'To request this support, please contact us at '
+                '[support@kobotoolbox.org](mailto:support@kobotoolbox.org)'
```

Comment:
If it's possible to reference one Constance setting from another, then could you replace this with `SUPPORT_EMAIL`, please? If not, maybe this string could have something like `##support email##` as a placeholder, and then replace that elsewhere(?)

Reply:
I chose the placeholder solution. Could not find a way to make it work as a reference